l5kit/l5kit/evaluation/csv_utils.py

```diff
@@ -173,14 +173,15 @@ def write_pred_csv(
     assert num_coords == 2
     assert timestamps.shape == track_ids.shape == (num_example,)
     assert confs is not None and confs.shape == (num_example, num_modes)
+    assert np.allclose(np.sum(confs, axis=-1), 1.0)
     assert num_modes <= MAX_MODES
 
     # generate always a fixed size json for MAX_MODES by padding the arrays with zeros
     coords_padded = np.zeros((num_example, MAX_MODES, future_len, num_coords), dtype=coords.dtype)
     coords_padded[:, :num_modes] = coords
     confs_padded = np.zeros((num_example, MAX_MODES), dtype=confs.dtype)
     confs_padded[:, :num_modes] = confs
-    # TODO check sum to 1
+
```

Comment:
How do we handle this now - the user is responsible for how they want to handle confidences - ie could be all 1s?

Reply:
are you reviewing the latest version? there is an assert on line 176 now

Reply:
Ah, found it below - see my comments there.